test/integration/driver.go

```diff
@@ -125,7 +125,6 @@ func setAuth(params infra) infra {
 	out := params
 	out.Name = "(auth infra)"
 	out.Auth = proxyconfig.ProxyMeshConfig_MUTUAL_TLS
-	out.Egress = false
```

Comment:
i'd like to keep those flags. we don't know if we need them again and they are not hard to maintain.